packages/tpetra/core/src/Tpetra_Details_computeOffsets.hpp

```diff
@@ -279,6 +279,32 @@ computeOffsetsFromCounts (const ExecutionSpace& execSpace,
        ": counts.size() = " << numCounts << " >= ptr.size() = " <<
        numOffsets << ".");
 
+#ifdef TPETRA_AVOID_PARALLEL_SCAN
```

Comment:
This really could have been a run-time switch.  I should have thought of that during yesterday's meeting.  That would have helped speed up debugging for the EMPIRE folks.